src/styles/syntax.css

```diff
@@ -0,0 +1,162 @@
+/**
+ * Gruvbox dark theme
+ *
+ * Adapted from a theme based on:
+ * Vim Gruvbox dark Theme (https://github.com/morhetz/gruvbox)
+ *
+ * @author Azat S. <to@azat.io>
+ * @version 1.0
+ */
+
+code[class*="language-"],
+pre[class*="language-"] {
+  color: var(--fg1);
+  /* fg1 / fg */
```

Comment:
I can probably get rid of all the comments that specify which color corresponds to which style